sentence_transformers/sparse_encoder/losses/SpladeLoss.py

```diff
@@ -43,13 +32,17 @@ def __init__(
 
         Args:
             model: SparseEncoder model
-            loss: The principal loss function to use (can be :class:`~sentence_transformers.sparse_encoder.losses.SparseMarginMSELoss`, :class:`~sentence_transformers.sparse_encoder.losses.SparseDistillKLDivLoss`,
-                       or :class:`~sentence_transformers.sparse_encoder.losses.SparseMultipleNegativesRankingLoss`)
-            lambda_corpus: Regularization weight for corpus (document) embeddings
-            lambda_query: Regularization weight for query embeddings
-            corpus_regulizer: The loss that will be used for the corpus. If not specify will be a FlopLoss.
-            query_regularizer: The loss that will be used for the query. If lambda_query is not 0 or None and this isn't specified, it will be a FlopLoss.
-
+            loss: The principal loss function to use can be any of the SparseEncoder losses except CSR related losses and flops loss.
+            lambda_corpus: Weight for the corpus regularization term. This term encourages sparsity in the document embeddings.
+                Will be applied to positive documents and all negatives one if some are provided.
+            lambda_query: Weight for the query regularization term. This term encourages sparsity in the query embeddings.
+                If None, no query regularization will be applied, it's not a problem if you are in an inference-free setup or
+                if you are having all_docs=True. Else you should have a lambda_query > 0.
+            all_docs: If True, all input embeddings are treated as documents and regularized together with lambda_corpus.
+                Espacially useful when training with input likes sentence pairs or more.
```

Comment:
```suggestion
            all_docs: If True, all input embeddings are treated as documents and regularized together with lambda_corpus.
                Especially useful when training with symmetric texts (e.g. pairs of documents) or more.
```
Maybe something like this?